process_report/process_report.py

```diff
@@ -230,6 +230,8 @@ def main():
 
     billable_projects = remove_non_billables(merged_dataframe, pi, projects)
     billable_projects = validate_pi_names(billable_projects)
+
+    backup_to_s3_old_pi_file(old_pi_file)
```

Comment:
you should check if `args.upload_to_s3` is set before trying to upload to s3 because otherwise if I make test runs I'd be uploading unnecessary stuff to S3 which I'd have to then clean up.